include/pybind11/detail/internals.h

```diff
@@ -15,7 +15,7 @@
 #    include "../gil.h"
```

Comment:
This one, too?

Reply:
Oops, didn't search for spaces after the `#`, thanks!

Reply:
This appeared to be the only one based on `git grep '# *include "\.\.'`.